testcases/kernel/syscalls/fallocate/fallocate03.c

```diff
@@ -162,9 +173,9 @@ void setup(void)
 	/* Create temporary directories */
 	TEST_PAUSE;
 
-	tst_tmpdir();
```

Comment:
As we discussed, put a comment here. Why we are removing tst_tmpdir() and creating folder with mkdir.